wandb/data_types.py

```diff
@@ -1101,15 +1164,39 @@ class Classes(Media):
     artifact_type = "classes"
 
     def __init__(self, class_set):
+        """Classes is holds class metadata intended to be used in concert with other objects when visualizing artifacts
+
+        Args:
+            class_set (list): list of dicts in the form of {"id":int|str, "name":str}
+        """
         super(Classes, self).__init__()
         self._class_set = class_set
         # TODO: validate
 
     @classmethod
     def from_json(cls, json_obj, source_artifact):
+        """Deserialize a `json_obj` into it's class representation. If additional resources were stored in the 
```

Comment:
Do we need to repeat this same comment everywhere?

Reply:
I thought, yes, but further testing shows no. incoming change